app/controllers/application_controller.rb

```diff
@@ -1,7 +1,7 @@
 class ApplicationController < ActionController::Base
-  protect_from_forgery prepend: true
-  before_action :authenticate_user!
-  before_action :configure_permitted_parameters, if: :devise_controller?
+  protect_from_forgery unless: -> { request.format.json? }
+  # before_action :authenticate_user!
+  # before_action :configure_permitted_parameters, if: :devise_controller?
```

Comment:
- [ ] [OPTIONAL] You have commented code present in your project which makes it hard to read the code. Please remove it.